modules/development/ssh.nix

```diff
@@ -29,6 +34,8 @@ in
 
     config = mkIf cfg.enable {
       services.openssh.enable = true;
+      services.openssh.extraConfig = pkgs.dbus-over-ssh.host.sshdConfigExtra;
```

Comment:
Please do not enable this for all targets, please move to targets/lenovo-x1-carbon.nix